src/components/molecules/timezone.tsx

```diff
@@ -0,0 +1,100 @@
+import React, { forwardRef, useCallback, useId, useMemo, useState } from "react";
+
+import { SingleValue } from "react-select";
+import TimezoneSelect from "react-timezone-select";
+
+import { getSelectDarkStyles, getSelectLightStyles } from "@constants";
+import { SelectOption, SelectProps } from "@interfaces/components";
+import { cn } from "@utilities";
+
+import { Hint } from "@components/atoms";
+
+const defaultTimezone = { label: "UTC", value: "UTC" };
+
+interface TimezoneSelectProps extends Omit<SelectProps, "options" | "value" | "onChange"> {
+	value?: string;
+	onChange?: (value: SingleValue<SelectOption>) => void;
+}
+
+export const AkTimezoneSelect = forwardRef<HTMLDivElement, TimezoneSelectProps>(
+	(
+		{
+			dataTestid,
+			disabled = false,
+			hint,
+			isError = false,
+			isRequired = false,
+			label = "Timezone",
+			onChange,
+			placeholder = "Select timezone",
+			value,
+			variant,
+			...rest
+		},
+		ref
+	) => {
+		const [selectedOption, setSelectedOption] = useState<any>(null);
```

Comment:
Don't use `any`, types help us to move fast and keep the code aligned :)
You should use the [SelectOption](https://github.com/autokitteh/web-platform/blob/ad8ea70769439fa96439247ea8436b77a30bc8d2/src/interfaces/components/forms/select.interface.ts#L24) to make it aligned to the Select component.